mmdeploy/backend/tensorrt/utils.py

```diff
@@ -54,8 +52,17 @@ def create_trt_engine(onnx_model: Union[str, onnx.ModelProto],
         >>>             device_id=0)
         >>>             })
     """
+
+    import os
+    old_cuda_device = os.environ.get('CUDA_DEVICE', None)
+    os.environ['CUDA_DEVICE'] = str(device_id)
+    import pycuda.autoinit  # noqa:F401
```

Comment:
we may add pycuda to requirements

Reply:
And I tested on Jetson, I only install successful with `pycuda==2019.1`. I haven't look up why yet. 
I suggest we do not using lastest version of pycuda, we can set it to `2019.1`

Reply:
Found that pycuda has been released not so often. Have you tried version of 2021.1?